Gymphony/src/app/core/components/navbar/navbar.component.ts

```diff
@@ -0,0 +1,17 @@
+import { Component } from '@angular/core';
+import { UserService } from '../../services/user.service';
+import { User } from '../../interfaces/user';
+
+
+@Component({
+  selector: 'app-navbar',
+  templateUrl: './navbar.component.html',
+  styleUrl: './navbar.component.css'
+})
+export class NavbarComponent {
+  public user?: User;
+
+  constructor(private userService: UserService) {
```

Comment:
Let's avoid writing code in components constructors, we use Angular lifecycle hooks instead